ch.theologeek.Manuskript.yml

```diff
@@ -0,0 +1,96 @@
+app-id: ch.theologeek.Manuskript
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+base: io.qt.qtwebengine.BaseApp
+base-version: "5.15-21.08"
+sdk: org.kde.Sdk
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    directory: texlive
+    subdirectories: true
+    autodelete: false
+    no-autodownload: true
+    version: '21.08'
+
+command: manuskript
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  #- --env=QT_QPA_PLATFORM=xcb
+  #- --env=TMPDIR=/var/tmp
+  # add paths of TeXlive Flatpak extension binaries
+  - --env=PATH=/usr/bin:/app/bin:/app/texlive/bin:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess # allow using the QtWebEngine
+
+cleanup:
+  - /lib/debug
```

Comment:
```suggestion
```

don't cleanup these

Reply:
why? I decreases the size of the Flatpak by 80MB

Reply:
not sure how you check. /lib/debug is separated by flatpak builder and put into it's own `.Debug` package

Reply:
I don't know how that would be useful, but I will keep it then.